lib/core/src/lib/card-view/components/card-view-selectitem/card-view-selectitem.component.stories.ts

```diff
@@ -0,0 +1,96 @@
+/*!
+ * @license
+ * Copyright 2019 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Meta, moduleMetadata, Story } from '@storybook/angular';
+import { CardViewSelectItemComponent } from './card-view-selectitem.component';
+import { CoreStoryModule } from './../../../testing/core.story.module';
+import { CardViewSelectItemModel, CardViewModule } from '../../../../..';
```

Comment:
Check if there is a nicer import 😄

Reply:
Done in ebbe77fc0dc50faec1ac02472bc23368adc8b240